staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2321,7 +2321,8 @@ type ResourceRequirements struct {
 	//
 	// This field is immutable.
 	//
-	// +listType=set
+	// +listType=map
+	// +listMapKey=name
```

Comment:
(you'll need to regen proto and openapi types for this change)

Reply:
Yep, but it looks good to me! I appreciate you asking if this is good before spending 30 minutes regening everything ;-)

Reply:
Right let me get working on the regen (first time on k/k in a while, wish me luck!)